viz.yaml

```diff
@@ -170,7 +170,7 @@ visualize:
   -
     id: futureSuitabilityFig
     location: figures/futureSuitabilityFig.svg
-    depends: processedFutureSuitability
+    depends: [processedFutureSuitability, siteTextData]
```

Comment:
I think you need to add siteTextData to the function signature for visualizeFutureSuitability?